.travis.yml

```diff
@@ -9,8 +9,8 @@ rvm:
   - 2.0.0
   - 2.1
   - 2.2
-  - 2.3.3
```

Comment:
Ummm, 2.3 is the same as 2.2 and 2.1, so that PATCH version can be omitted. It would preferable to unify the description of PATCH version about them.